services/distribution/src/main/java/app/coronawarn/server/services/distribution/dgc/client/ProdDigitalCovidCertificateClient.java

```diff
@@ -67,12 +67,20 @@ public ValueSet getValueSet(String hash) throws FetchValueSetsException {
         FetchBusinessRulesException::new);
   }
 
+  @Override
+  public List<BusinessRuleItem> getBnRules() throws FetchBusinessRulesException {
+    return getResponseAndTreatExceptions(digitalCovidCertificateClient::getBnRules,
+        "business rules",
+        FetchBusinessRulesException::new);
+  }
+
   @Override
   public BusinessRule getCountryRuleByHash(String country, String hash) throws FetchBusinessRulesException {
-    return getResponseAndTreatExceptions(
+    BusinessRule businessRule =  getResponseAndTreatExceptions(
```

Comment:
Can we directly return here?

Reply:
was used for debugging, need to revert, thanks